website/playground/index.js

```diff
@@ -372,6 +375,7 @@ class URLParams {
         window.location.pathname
       }?${this.params.toString()}`;
       window.history.replaceState({ path: url }, "", url);
+      localStorage.setItem(STORAGE_KEY_CODE, code);
```

Comment:
setItem and getItem might throw errors depending on certain browsers. It might be good to wrap them around try/catch blocks and handle those edge cases, regarding their probability of occurance.